test/integration/usage-reports.v4.test.js

```diff
@@ -0,0 +1,307 @@
+/* eslint-disable no-console */
+/**
+ * (C) Copyright IBM Corp. 2020.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+const UsageReportsV4 = require('../../dist/usage-reports/v4');
+const { readExternalSources } = require('ibm-cloud-sdk-core');
+const authHelper = require('../resources/auth-helper.js');
+
+// testcase timeout value (60s).
+const timeout = 60000;
+
+// Location of our config file.
+const configFile = 'usage_reports.env';
+
+const describe = authHelper.prepareTests(configFile);
+
+let usageReportsService;
+let config;
+let accountId;
+let resourceGroupId;
+let orgId;
+let billingMonth;
+
+describe('UsageReportsV4_integration', () => {
+  jest.setTimeout(timeout);
+
+  beforeAll(async () => {
+    usageReportsService = UsageReportsV4.newInstance({});
+
+    config = readExternalSources(UsageReportsV4.DEFAULT_SERVICE_NAME);
+
+    expect(usageReportsService).not.toBeNull();
+    expect(config).not.toBeNull();
+
+    accountId = config.accountId;
+    resourceGroupId = config.resourceGroupId;
+    orgId = config.orgId;
+    billingMonth = config.billingMonth;
+    expect(accountId).not.toBeNull();
+    expect(resourceGroupId).not.toBeNull();
+    expect(orgId).not.toBeNull();
+    expect(billingMonth).not.toBeNull();
+
+    // console.log('Finished setup.');
+  });
+
+  afterAll(async () => {
```

Comment:
Is this generated? Why include the function if it's not needed?

Reply:
no it's not... I copied the beforeAll and afterAll sections from another test and ended up not needed the afterAll section.  I'll remove that.